Default.sublime-keymap

```diff
@@ -0,0 +1,3 @@
+[
+    { "keys": ["super+shift+o"], "command": "side_bar_folders_quick_switch" }
```

Comment:
I have added instructions to the readme for changing the shortcut. Are you suggesting that I should  remove the keymap file all together and leave it up to the user to configure the shortcut after installing the plugin?